openapi.yml

```diff
@@ -3442,14 +3442,12 @@ paths:
       - in: query
         name: workflow_state
         schema:
-          type: string
-          enum:
-          - DONE
-          - NEW
-          - PRE_SECONDARY_ASSESSMENT
-          - REJECTED
-          - SECONDARY_ASSESSMENT
-          - TRIAGE
+          type: array
+          items:
+            type: string
+        description: Multiple values may be separated by commas.
```

Comment:
Nitpick: The enum in openapi.yml seems kind of useful for knowing which values are valid. We are losing that usefulness here. I see that Flaw's `cve_id` has the same drawback in openapi.yml, so this PR already has a precedent that it is OK. I'd just like to discuss (even after merging this) whether we want to somehow improve openapi.yml for fields that use `CharInFilter`, and whether anyone would even care. @RedHatProductSecurity/osidb-devs thoughts?

Reply:
For `cve_id` I think it is kinda different as it is not an enum and the field is pretty self explanatory, but loosing and enum values might be problematic for anyone who would like to work with the API. I posted some possible solutions below, not tested them thoroughly or sure about them, just something to think about.